lib/luatest_server.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, *args, **kwargs):
     def execute(self, server):
         """Execute test by luatest command
 
-        Execute `luatest -c --no-clean --verbose <name>_test.lua --output tap`
-        command. Disable capture mode and deletion of the var directory.
-        Provide a verbose output in the tap format. Extend the command by
-        `--pattern <pattern>` if the corresponding option is provided.
+        Execute `luatest -c --no-clean -l <path> --verbose <name>_test.lua --output tap`
```

Comment:
Please provide the full name of the option.

Reply:
Fixed